README.md

```diff
@@ -38,7 +38,8 @@ Then:
 ## Additional styles
 
 📦 [Install the StackOverflow Syntax Themes usercss](https://raw.githubusercontent.com/StylishThemes/Feature-Override-Styles/master/stackoverflow-syntax-themes.user.css)<br>
-This style allows you to choose preset syntax themes for StackOverflow and associated sites.<br><br>
+- This style allows you to choose preset syntax themes for StackOverflow and associated sites.
```

Comment:
```suggestion

```
I forgot about this sentence. Needs to be shortened the whole lot I think. Less reading better for all users apparently.